build.sbt

```diff
@@ -87,7 +65,13 @@ lazy val commonSettings = Seq(
   organization := "com.typesafe.play",
   scalaVersion := scala212,
   crossScalaVersions := Seq(scala213, scala212),
-  scalacOptions in (Compile, doc) ++= scalacOptionsFor(scalaBinaryVersion.value),
+  scalacOptions ++= scalacOpts,
+  scalacOptions in (Compile, doc) ++= Seq(
+    "-Xfatal-warnings",
+
+    // Work around 2.12 bug which prevents javadoc in nested java classes from compiling.
```

Comment:
I think this should be fixed now.

Reply:
No looks like it's not:

```
[error] /home/travis/build/playframework/play-ws/play-ahc-ws-standalone/src/main/java/play/libs/oauth/OAuth.java:20:18: not found: type ServiceInfo
[error]     public OAuth(ServiceInfo info) {
[error]                  ^
```

Reply:
https://github.com/scala/bug/issues/11365